cpanfile

```diff
@@ -57,13 +58,15 @@ requires 'Moo', '2.000002';
 requires 'MooseX::ClassAttribute';
 requires 'MooseX::Role::Parameterized', '1.02';
 requires 'MooseX::StrictConstructor';
+requires 'MooseX::Fastly::Role';
```

Comment:
Maybe we could make it a habit to include minimum version numbers when adding to the `cpanfile`?